Automattic/wp-calypso
# Display FAQ section in the new pricing page.

#### Proposed Changes

* Add `StoreFooter` in the new `ProductStore` component to display FAQ section. 

#### Testing Instructions



* Checkout out the PR by running `git fetch && git checkout add/pricing-page-faq-section`
* Set `jetpack/pricing-page-rework-v1`  to `true` in `config/jetpack-cloud-development.json` if it is not yet set.
* Run `yarn start-jetpack-cloud`
* Goto http://jetpack.cloud.localhost:3000/pricing
*  Confirm that you see the FAQ section in the new pricing page.

<img width="1271" alt="Screen Shot 2022-08-22 at 8 16 13 PM" src="https://user-images.githubusercontent.com/56598660/185919040-b774a013-f1aa-4f6f-90ab-08ffb470c0a0.png">



#### Pre-merge Checklist



- [x] [Have you written new tests](https://wpcalypso.wordpress.com/devdocs/docs/testing/index.md) for your changes (no test needed)?



Related to 1202796695664022-as-1202838239202320

---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - 0-as-1202838239202320
